preset/postgres/preset_test.go

```diff
@@ -71,11 +78,18 @@ func TestPreset_withDefaults(t *testing.T) {
 	connStr := fmt.Sprintf(
 		"host=%s port=%d user=%s password=%s  dbname=%s sslmode=disable",
 		container.Host, container.DefaultPort(),
-		"gnomock", "gnomick", "mydb",
+		"postgres", "password", "postgres",
```

Comment:
Nice, I forgot why it was something else before😼
Now I don't understand why it worked...